### ISSUE TITLE
Can't see over some natural wonders

### ISSUE THREAD

**Issue by Phalanx1** [LoneGazebo/Community-Patch-DLL#3341]
_1. Mod version (i.e Date - 4/23):_
6/22

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
This bug has already been fixed for Lake Victoria (#2501), but some wonders like Old Faithful still block vision, even when the unit is on hills, despite not being a mountain wonder. I think this issue is also present on Fountain of Youth, King Solomon's Mines and El Dorado.
Also, as far as I know the Incas still have to spend more that one move to go through mountain natural wonders, despite only using one move to go through mountains. Working as intended?

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
I believe that Natural Wonders are partially hardcoded to be mountains in some aspects, unfortunately. @ilteroi is this correct?

**Comment by Phalanx1** [comment]
Are you sure about that? You seem to have found a solution for the problem with Lake Victoria in the issue I linked on the OP.
About the issue I mentioned on the Inca UA and mountain wonders, is it possible to alter a wonder's movement cost? (By the way, I'm not 100% sure if Inca Units can cross mountain wonders in the current version. I noticed this second issue some time ago, but just got to post it now, so I might be remembering something wrong.)
